pkg/controllers/offline.go

```diff
@@ -140,6 +141,7 @@ func (s *sccOfflineMode) ReconcileActivateError(registrationObj *v1.Registration
 	v1.RegistrationConditionActivated.False(registrationObj)
 	v1.RegistrationConditionActivated.Reason(registrationObj, "offline activation failed")
 	v1.RegistrationConditionOfflineCertificateReady.SetError(registrationObj, "cannot validate offline certificate", activationErr)
+	registrationObj.SetCurrentCondition(v1.RegistrationConditionOfflineCertificateReady)
```

Comment:
Same question here, do we need to update on 136?

Reply:
Also covered by `PrepareSuccessfulActivation`